### DISCUSSION TITLE
Problem FCPX

### DISCUSSION THREAD

**Discussion by Mavision57** [latenitefilms/GyroflowToolbox#37] [Support]
<img width="910" alt="Capture d'écran 2024-11-07 à 10 03 01" src="https://github.com/user-attachments/assets/898eeed9-3843-419d-bfb4-35cab3c01cda">

I have a problem with FCPX on apple silicon M1 PRO please

**Comment by latenitefilms** [comment]
I'm not really sure why you have two Gyroflow Toolbox Effects? Maybe delete both these Motion Templates from your Movies folder, and reinstall from the Gyroflow Toolbox application from the Mac App Store?

**Reply by Mavision57** [reply]
I'm sorry, I'm French, so I downloaded the ZIP file for free from the website. I copied the entire folder into the "Motion templates/effect" folder, but it caused the error shown in the image. I then tried copying the other folder named "Source/Gyroflow/" into "Motion templates/effect," which is why I now have two "Toolbox effects," but neither of them is working.
<img width="281" alt="Capture d’écran 2024-11-07 à 11 19 29" src="https://github.com/user-attachments/assets/212153a1-de55-4fb6-9e79-13633435f3e0">

**Reply by latenitefilms** [reply]
There's no free version on the website. You either need to install it from the Mac App Store, or compile it yourself using Xcode.
